__tests__/app.test.js

```diff
@@ -86,3 +86,30 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an array of all comments on an article sorted by most recent", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments[0]).toEqual(
```

Comment:
Good start however your description says array and sorted, but we're only checking the first object is correct, not the rest of the array - and we haven't checked they're sorted!